### PR TITLE
Setup basic Dagger config PREF-31

### PR DESCRIPTION
**Pull request by IwoHerka**

### CODE REVIEW

File: buildSrc/src/main/java/Dependencies.kt [hunk 1]
```diff
@@ -26,7 +26,10 @@ object Config {
      */
     val buildValues = listOf(
         "String", "API_URL",      "some.api.com/",
-        "String", "API_PROTOCOL", "https"
+        "String", "API_PROTOCOL", "https",
+
+        "long", "HTTP_READ_TIMEOUT", "60",
```
Comment (rkowalski4) [inline buildSrc/src/main/java/Dependencies.kt:31]: Can we move `60` to some constant?

Comment (IwoHerka) [reply buildSrc/src/main/java/Dependencies.kt:31]: Where would you like to put it?

Comment (rkowalski4) [reply buildSrc/src/main/java/Dependencies.kt:31]: I'm not sure. What about object property/val - something like versions in Version object. Maybe extra object with constants?

Comment (IwoHerka) [reply buildSrc/src/main/java/Dependencies.kt:31]: I'm not fond of this to be honest. That's 3 levels of indirection. I would leave build config as it is.

**Review by rkowalski4 — APPROVED** [review]

**Pull request merged**